pages/docs/user-docs/docs-environment-metadata.md

```diff
@@ -6,36 +6,40 @@ folder: docs
 toc: false
 ---
 
-Singularity containers support environment variables and labels that can be added by user during the bootstrap process.
+Singularity containers support environment variables and labels that you can add to your container during the build process.
 
 {% include toc.html %}
 
 ## Environment
 
-If you are importing a Docker container, the environment will be imported as well. If you want to define custom environment variables in your bootstrap recipe file `Singularity` you can do that like this
+If you build a container from Singularity Hub or Docker Hub, the environment will be imported into the container at build time. You can also define custom environment variables in your Recipe file like so:
 
```

Comment:
This doesn't make sense to me. Singularity Hub containers already have an environment, how is it imported? You are just pulling and using the previous container.

Reply:
How about "the environment will be included with the container at build time" to cover both instances.

Reply:
that would work!